.github/CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ Please take a moment to review this document in order to make the contribution
 process easy and effective for everyone involved.
 
 Following these guidelines helps to communicate that you respect the time of
-the developers managing and developing this open source project. In return,
+the developers managing and developing this open-source project. In return,
```

Comment:
"open source project" is correct. matter of preference if you want to add a hyphen or not, but it's commonly used without